auth/oidc.go

```diff
@@ -0,0 +1,339 @@
+// Copyright © 2019 Banzai Cloud
```

Comment:
This file has been moved here from banzaicloud/auth, I would like to degrade that package entirely, this is one step forward.